src/DateTime/tests/test_datetime.py

```diff
@@ -39,7 +39,7 @@
 
 try:
     __file__
-except NameError:
+except NameError:   # pragma: no cover
     f = sys.argv[0]
 else:
     f = __file__
```

Comment:
I am not sure if this dance is any more needed at all. But I think that's not part of this PR.